.github/workflows/kind.yml

```diff
@@ -134,16 +134,23 @@ jobs:
         deployOnHub: ${{ matrix.deployOnHub }}
         HUB_ONLY: ${{ inputs.hub_only_component }}
         HOSTED: ${{ matrix.hosted }}
+        REPO: ${{ github.event.repository.name }}
       run: |
         export MANAGED_CONFIG=${MANAGED_KUBECONFIG}
         export HUB_CONFIG=${HUB_KUBECONFIG}
         export HUB_CONFIG_INTERNAL=${HUB_INTERNAL_KUBECONFIG}
 
         if [[ "${HOSTED}" == "hosted" ]]; then
           export WATCH_NAMESPACE=cluster2-hosted
-          export KIND_NAMESPACE=cluster2-hosted
           export KIND_NAME=policy-addon-ctrl1
           export KUBECONFIG=${HUB_KUBECONFIG}
+          export KIND_NAMESPACE=cluster2-hosted
+          if [[ "${REPO}" == "governance-policy-propagator" ]]; then
```

Comment:
Instead of this workflow choosing the namespace, could the repo calling this workflow explicitly override the namespace? In other words, something in the `governance-policy-propagator` GitHub workflow file explicitly sets it.

This would be cleaner in my opinion.

Reply:
```
      if [ ! "${KIND_NAMESPACE}" ]; then
            export KIND_NAMESPACE=cluster2-hosted
       fi
```
Then I will add `KIND_NAMESPACE `env at propagator repo/framework-kind.yaml
How about this?

Reply:
We should be able to use `HUB_ONLY` to conditionally set the namespace to `open-cluster-management` for hub components. Would that cover it?

Reply:
I think it is better set env in each repo's makefile,  config, iam, cert vs propagator, addon framework-kind.yaml(which is in each repo) don't have information about hosted mode so

Reply:
I added like this in each repo.
```
	if [ "$(HOSTED)" -eq "hosted" ]; then\
		export KIND_NAMESPACE=open-cluster-management ;\
	else 
		$(MAKE) kind-deploy-controller;\
	fi;
```

Reply:
I'm not so sure. The "cluster2-hosted" name is defined this this repo, so it seems that it makes more sense to set it here also rather than in each repo?

Reply:
In propagator controller KIND NAMESPCE should be` open-cluster-management` and addon controller should be `governance-policy-addon-controller-system `